docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -342,3 +342,21 @@ _view=sumologic_slo_output sloId="<your-SLO-ID>"
 ```
 
 These log messages will be delayed by one hour, as the system ensures consistency to account for ingest delay of source telemetry.
+
+
+### Create an SLO Lookup Table
+
+If you have an existing [Lookup Table](/docs/search/lookup-tables) you can enrich it detailed SLO data (such as sloName, signalType, sliQueryType, sloDescription) using the `_view=sumologic_slo_output` snippet.
```

Comment:
@kimsauce the SLO Lookup Table is a different feature. It's basically an internal Sumo managed lookup table (i.,e the user does not have to create or manage this lookup table). It resides under a fixed path `sumo://content/slos` and contains the metadata of all the SLOs in the user's environment.

In the docs, we can describe that there are two ways to use this broadly:
1. `_view=sumologic_slo_output | lookup * from sumo://content/slos on sloId, sloVersion` - joins the results of the SLO precomputed data from _view = sumologic_slo_output with the metadata contained in the internal lookup table based on the joining key (sloId, sloVersion)
2. `cat sumo://content/slos` enlists the contents of the lookup table

Reply:
I think the current content here and below is around the `Open In` (drilldown action).

Reply:
Thank you! I'll tag you in the revision to re-review